homeassistant/components/adax/climate.py

```diff
@@ -152,6 +156,18 @@ def __init__(self, adax_data_handler: AdaxLocal, unique_id: str) -> None:
             manufacturer="Adax",
         )
 
+    async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
+        """Set hvac mode."""
+        if hvac_mode == HVACMode.HEAT:
+            temperature = max(
+                self.min_temp, self._attr_target_temperature or self.min_temp
+            )
+            await self._adax_data_handler.set_target_temperature(temperature)
+        elif hvac_mode == HVACMode.OFF:
+            await self._adax_data_handler.set_target_temperature(0)
```

Comment:
@gjohansson-ST Here you can see how the heater is turned off, by setting target temperature 0. This is simply how the local API of adax works. The way I discovered this was by physically turning of the temperature on the device (by pressing down arrow until the display reads "-") and discovering that it was returning target temperature 0.